apps/ledger-live-desktop/tests/models/SettingsPage.ts

```diff
@@ -8,6 +8,13 @@ export class SettingsPage {
   readonly experimentalTab: Locator;
   readonly experimentalDevModeToggle: Locator;
   readonly carouselSwitchButton: Locator;
+  readonly counterValueSelector: Locator;
+  readonly counterValueSearchBar: Locator;
+  readonly counterValuedropdownChoiceEuro: Locator;
```

Comment:
change to `counterValueDropdownChoiceEuro`

Reply:
The d is still not capitalised

Reply:
i'm not push any commit yet, will do it later😅